1000-accordion-design-guidance.md

```diff
@@ -0,0 +1,29 @@
+---
+category: component design guidance
+state: draft
+created: 2024-10-15
+updated: 2024-10-22
+---
+
+# Accordion Design Guidance
+
+An [accordion](https://clarity.design/documentation/accordion) is a collection of vertically stacked sections with multiple content panels.
+
+## Guidance
+
+- **Must** use accordions when you need to display a lot of different information and the user doesn’t need it all at once.
+- **Must not** use an accordion if all the content is important to a user.
```

Comment:
```suggestion
- Design teams **must not** use an accordion when all content is important to the user, as it may cause confusion or hinder access to essential information.
```